turbinia/client.py

```diff
@@ -710,6 +731,90 @@ def format_task_statistics(
     report.append('')
     return '\n'.join(report)
 
+  def format_worker_status(
+      self, instance, project, region, days=0, all_fields=False):
+    """Formats the recent history for Turbinia Workers.
+
+    Args:
+      instance (string): The Turbinia instance name (by default the same as the
+          INSTANCE_ID in the config).
+      project (string): The name of the project.
+      region (string): The name of the zone to execute in.
+      days (int): The number of days we want history for.
+      all_fields (bool): Include historical Task information for the worker.
+    Returns:
+      String of Request status
+    """
+    # Set number of days to retrieve data
+    num_days = 7
+    if days != 0:
+      num_days = days
+    task_results = self.get_task_data(instance, project, region, days=num_days)
+    if not task_results:
+      return ''
+
+    # Sort task_results by last updated timestamp.
+    task_results = sorted(
+        task_results, key=itemgetter('last_update'), reverse=True)
+
+    # Create dictionary of worker_node: {{task_id, task_update,
+    # task_name, task_status}}
+    workers_dict = {}
+    for result in task_results:
+      worker_node = result.get('worker_name')
+      status = result.get('status')
+      status = status if status else 'No task status'
+      if worker_node and worker_node not in workers_dict:
+        workers_dict[worker_node] = []
+      if worker_node:
+        task_dict = {}
+        task_dict['task_id'] = result.get('id')
+        task_dict['last_update'] = result.get('last_update')
+        task_dict['task_name'] = result.get('name')
+        task_dict['status'] = status
+        # Check status for anything that is running.
+        if 'running' in status:
+          run_time = (datetime.now() -
+                      result.get('last_update')).total_seconds()
+          run_time = timedelta(seconds=run_time)
+          task_dict['run_time'] = run_time
+        else:
+          task_dict['run_time'] = result.get('run_time')
+        workers_dict[worker_node].append(task_dict)
+    # Generate report header
+    report = []
+    report.append(
+        fmt.heading1(
+            'Turbinia report for Worker activity within {0:d} days'.format(
+                num_days)))
+    report.append(
+        fmt.bullet('{0:d} Worker(s) found.'.format(len(workers_dict.keys()))))
+    for worker_node, tasks in workers_dict.items():
+      report.append('')
+      report.append(fmt.heading2('Worker Node: {0:s}'.format(worker_node)))
+      # Append the statuses chronologically
+      run_status, queued_status, other_status = [], [], []
+      for vals in tasks:
```

Comment:
Extremely minor nit, but I think our style guide has something about variable names being fully written out when possible (also, it could potentially just be `task` instead of `vals`).

Reply:
Updated to `task` instead of `vals`